lib/beaker/cli.rb

```diff
@@ -46,6 +46,7 @@ def initialize
 
       @options[:log_dated_dir] = Beaker::Logger.generate_dated_log_folder(@options[:log_dir], @timestamp)
       @options[:xml_dated_dir] = Beaker::Logger.generate_dated_log_folder(@options[:xml_dir], @timestamp)
+      @options[:logger_provisioning] = Logger.new(File.join(@options[:log_dated_dir], @options[:log_provisioning]), { :quiet => true })
```

Comment:
Why is this `Logger` call not prefixed with `Beaker::Logger` like the previous 2 lines?